INTERPRETER.s

```diff
@@ -1337,7 +1337,7 @@ UPPOS		XCH	L		# SAVE DECREMENTED UPPER PART.
 		TS	A		# SKIPS ON OVERFLOW
 		TCF	+2
 		INCR	L		# RESTORE UPPER TO ORIGINAL VALUE
-		XCH	L		# SWAP A + L BANCK.
+		XCH	L		# SWAP A + L BANK.
```

Comment:
Both are wrong.

`BACK` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1148.jpg)